src/frontend-proxy.ts

```diff
@@ -0,0 +1,95 @@
+import { getSubdomain, getPathname } from './url-utils'
+
+export const FRONTEND_DOMAIN = 'frontend-sooty-ten.now.sh'
+const API_ENDPOINT = 'https://api.serlo.org/graphql'
```

Comment:
Should be dependent on the environment so we can test out a new API version. You might even think about sending that down to the frontend as a header so that the frontend uses a different API for different environments.

Reply:
- [ ] API-ENDPOINT als Header weitergeben